static/docs/user-guide/contributing-documentation.md

```diff
@@ -105,12 +81,13 @@ follow the steps below:
 - **Markdown and JS files:** content must be properly formatted at 80 symbols
   width. We recommend using Visual Studio Code and the
   [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)
-  plugin to format the content of Markdown files or install the pre-commit hook
-  described above.
+  plugin to format the content of Markdown files or it will be done by a
+  pre-commit hook which was automatically integrated when we ran `npm install`.
 
 - **Markdown and JS files:** we use `prettier` default conventions to format our
-  files. It's highly recommended to install the hook or run `prettier` manually
-  before submitting the pull request.
+  files. The formatting of staged files will be automatically done by the
+  pre-commit hook we have configured or run `prettier` manually before
```

Comment:
"...we have configured. You may also run `npx prettier --write {file path(s)}` manually before committing changes." (suggestion)

Reply:
@jorgeorpinel This reminds me where does [this issue](https://github.com/iterative/dvc.org/issues/364) fixes takes place?